float32 waypoint_x
float32 waypoint_y
---
float32 time_s
---
float32 distance_to_waypoint